_includes/payment-order-other-features.md

```diff
@@ -231,6 +231,74 @@ Content-Type: application/json
 }
 ```
 
+{% if include.documentation_section == "payment-menu" %}
+
+## Payer Aware Payment Menu
```

Comment:
Should `Payer Aware Payment Menu` be added in this PR? I thought @marinafrydenb added that in #598?

Reply:
No it should be a part of this PR. Most likely a merging mishap or something like that.